python/ray/autoscaler/v2/instance_manager/reconciler.py

```diff
@@ -1167,6 +1236,332 @@ def _scale_cluster(
 
         Reconciler._update_instance_manager(instance_manager, version, updates)
 
+    @staticmethod
```

Comment:
remove the `@staticmethod` as it is just a helper function when adding `def _scale_virtual_clusters`